frontend/ui/widget/buttondialogtitle.lua

```diff
@@ -38,8 +39,8 @@ function ButtonDialogTitle:init()
                 ges = "tap",
                 range = Geom:new{
                     x = 0, y = 0,
-                    w = Screen:getWidth(),
-                    h = Screen:getHeight(),
+                    w = self.modal and 0 or Screen:getWidth(),
+                    h = self.modal and 0 or Screen:getHeight(),
```

Comment:
Is `modal=true `to be used when we don't want Tap outside to close ?
We have this in confirmbox.lua and multiconfirmbox.lua:
`dismissable = true, -- set to false if any button callback is required`
which does that by not registering the gesture (instead of registering a 0x0 gesture like you do here?)

Reply:
`dismissable` is very nice option. I will use it in buttondialogtitle widget.